docs/hub/models-libraries.md

```diff
@@ -24,6 +24,7 @@ The table below summarizes the supported libraries and their level of integratio
 | [RL-Baselines3-Zoo](https://github.com/DLR-RM/rl-baselines3-zoo) | Training framework for Reinforcement Learning, using [Stable Baselines3](https://github.com/DLR-RM/stable-baselines3).| ❌ | ✅ | ✅ | ✅ |
 | [Sentence Transformers](https://github.com/UKPLab/sentence-transformers)    | Compute dense vector representations for sentences, paragraphs, and images.          | ✅ | ✅ | ✅ | ✅ |
 | [spaCy](https://github.com/explosion/spaCy)                                 | Advanced Natural Language Processing in Python and Cython.                           | ✅ | ✅ | ✅ | ✅ |
+| [Scikit Learn](https://github.com/skops-dev/skops)                          | Machine Learning in Python.                           | ✅ | ✅ | ✅ | ✅ |
```

Comment:
Probably better to link to the docs instead: https://skops.readthedocs.io/en/stable/

And I'd probably write it as "scikit-learn with skops" instead of "Scikit Learn" maybe?

Reply:
Good idea, done!